packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+*No Content is returned when the deactivation is successful.*
+
+## Rule Operations
+
+### Get Rules in a Rule Set
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/rules" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
+
+### Create a rule
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/rules" />
+
+##### Request Parameters
+
+The policy ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "Okta:SignOn",
+  "name": "Example Rule",
+  "status": "ACTIVE",
+  "default": false,
+  "priority": 0,
+  "conditions": [
+     {
+        "key": "Okta:User",
+        "op": "IN_LIST",
+        "value": [
+           "00u1pdsiceJZLRJMSEUA"
+        ]
+     }
+  ],
+  "action": "ALLOW",
+  "requirement: [
+     "verificationMethod": {
+         "type": "ASSURANCE",
+         "factorMode": "1FA",
+         "constraints": [
+            {
+              "knowledge": {
+                 "types": [
+                   "password"
+                 ]
+              }
+            }
+         ],
+         "reauthenticateIn": "PT1M"
+     }
+  ]
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
+
+### Delete a rule
+
+<ApiOperation method="delete" url="/api/v1/policies/${rulesetId}/rules/${ruleId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content*
+
+### Get a rule
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/rules/${ruleId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rules object](#rules-object)
+
+### Update a rule
```

Comment:
Which properties can or can't be updated via API?

Reply:
Can:
* Name
* Status (non default)
* Priority
* Conditions (non default)
* Requirement

Reply:
What do you mean by "(non default)"?

Reply:
The `default` property can't be updated?

Reply:
Correct